src/components/ItemSelection/ItemSelection.tsx

```diff
@@ -66,6 +66,7 @@ const ItemSelection: FC<ItemSelectionProps> = ({
           <FormButton
             theme="formButtonGreen"
             text={getButtonTextFromType(type)}
+            fullWidth
```

Comment:
Is this just to set it to true? If so maybe do `fullWidth={true}` just for clarity